ci/ray_ci/windows_container.py

```diff
@@ -0,0 +1,33 @@
+import subprocess
+import sys
+from typing import List, Optional
+
+from ci.ray_ci.container import Container
+
+
+class WindowsContainer(Container):
+    def install_ray(self, build_type: Optional[str] = None) -> List[str]:
+        assert build_type is None, f"Windows does not support build types {build_type}"
+        subprocess.check_call(
+            [
+                "docker",
+                "build",
+                "-t",
+                self._get_docker_image(),
+                "-f",
+                "c:\\workdir\\ci\\ray_ci\\tests.windows.env.Dockerfile",
```

Comment:
> where is this file? I do not see it in this PR?

It's not used until the top of this stack https://github.com/ray-project/ray/pull/41640; i can move the implementation up there if you want

> it would be nicer if we can support using a different tag for source and dest image.

It's been used in multiple places now with the assumption that the source and dest image name is the same https://github.com/ray-project/ray/blob/master/.buildkite/others.rayci.yml#L32; https://github.com/ray-project/ray/blob/master/.buildkite/core.rayci.yml#L57. Specifying different names will require some non-simple changes throughout.